geoh5py/groups/uijson.py

```diff
@@ -121,7 +107,7 @@ def copy(
         parent=None,
         *,
         copy_children: bool = False,
-        copy_relatives: bool = True,
+        copy_relatives: bool = False,
```

Comment:
That won't work because the nested UIJGroup won't copy theirs

<img width="840" height="613" alt="image" src="https://github.com/user-attachments/assets/cbb8d648-27c8-4f6e-8aeb-600475b7836e" />